infra/monitoring/main.tf

```diff
@@ -58,21 +58,25 @@ data "aws_iam_policy_document" "push_metric_policy" {
 }
 
 resource "aws_iam_policy" "push_metric_policy" {
-  name        = "push-metric-policy"
+  name        = var.module_name == "hudi" ? "push-metric-policy-hudi" : "push-metric-policy-json"
```

Comment:
Instead of making a condition here. You can just make a formatted string and take the` output_format` variable, instead of a new module_name variable (which also always seems to be  the same value as output_format)

Reply:
name = "push-metric-policy-${output_format}"